addons/vol-slider/userstyle.css

```diff
@@ -42,13 +47,19 @@
   width: 12px;
   height: 12px;
   border-radius: 50%;
-  background-color: #cf63cf;
+  background-color: #c234c2;
 }
 
 .sa-vol-slider-input::-moz-range-thumb {
   width: 12px;
   height: 12px;
   border: none;
   border-radius: 50%;
-  background-color: #cf63cf;
+  background-color: #c234c2;
+}
+
+.sa-vol-slider:not(:hover) > .sa-vol-slider-input {
+  opacity: 0;
+  width: 0;
+  margin-left: -3px;
```

Comment:
Will this be slightly annoying for touch screen users? Is that a problem?

Reply:
What about making it a setting?

Reply:
> What about making it a setting?

Could always do that. I was thinking about it.